docs/concept-exercises.md

```diff
@@ -0,0 +1,74 @@
+# Concept Exercises
+
+Concept Exercises replace V2's Core Exercises. They are exercises designed to teach specific programming concepts.
```

Comment:
```suggestion
Concept Exercises replace V2's Core Exercises. They are exercises designed to teach specific (programming) concepts.
```

I think we only need to imply that we are teaching programming concepts once, as that is likely obvious to most.